### PR TITLE
refactor: log when loading config from file

### PR DESCRIPTION
**Pull request by weihanglo**
When I was tracing some nasty bug and wanted to point at config probding, I wish there were some logs standing out and telling me I was wrong.

Would be great if every config file loading is logged.

### CODE REVIEW

File: src/cargo/util/config/mod.rs [hunk 1]
```diff
@@ -1200,6 +1200,8 @@ impl Config {
                 path.display()
             );
         }
+        tracing::event!(tracing::Level::DEBUG, ?path, ?why_load, includes, "load config from file");
```
Comment (epage) [inline src/cargo/util/config/mod.rs:1203]: nit: can't we do `tracing::debug!(` and skip referencing the constant?

**Review by epage — APPROVED** [review]
Feel free to r= me whichever way you go

File: src/cargo/util/config/mod.rs (hunk 1 above)
Comment (weihanglo) [reply src/cargo/util/config/mod.rs]: Nice catch!

**Comment by weihanglo** [comment]
@bors r+

Thanks for the review!

**Pull request merged**